.github/workflows/kernel.yml

```diff
@@ -71,7 +71,7 @@ jobs:
             SUBTARGET=$(echo $TARGET_SUBTARGET | cut -d "/" -f 2)
 
             if echo "$CHANGED_FILES" | grep -q target/linux/generic ||
-              echo "$CHANGED_FILES" | grep -q "target/linux/$TARGET"; then
```

Comment:
This was OK.

Reply:
In other words, this change would only work for `lantiq` target, as there is only `package/kernel/lantiq` `$TARGET`  package. It would again fail to cover other kernel packages.